nodejs/awsx/apigateway/README.md

```diff
@@ -98,6 +98,82 @@ A complete example can be found [here](https://github.com/pulumi/pulumi-awsx/blo
 
 API Gateway can perform basic validations against request parameters, a request payload or both. When a validation fails, a 400 error is returned immediately.
 
+### Lambda Authorizers
+
+[Lambda Authorizers](https://docs.aws.amazon.com/apigateway/latest/developerguide/apigateway-use-lambda-authorizer.html) are AWS Lambda functions that provide control access to an API. You can define a Lambda Authorizer for an Event Handler Route or a Static Route.
+
+To define an Authorizer, you provide a Lambda that fulfills `aws.lambda.EventHandler<AuthorizerEvent, AuthorizerResponse>` or you provide information on a pre-existing Lambda authorizer. The exmaple below shows defining the Authorizer Lambda directly inline. See the [Event Handler Route](#Event-Handler-Route) section on other ways you can define a Lambda for the Authorizer.
+
```

Comment:
exmaple !

Reply:
okay -- i was worried about "over exampling" but i can add these :)

Reply:
lol ohh i have a typ youre not telling me to write more examples haha